src/services/codefixes/helpers.ts

```diff
@@ -386,11 +387,13 @@ export function createSignatureDeclarationFromSignature(
             }
         }
         const newParameters = sameMap(parameters, parameterDecl => {
-            const importableReference = tryGetAutoImportableReferenceFromTypeNode(parameterDecl.type, scriptTarget);
-            let type = parameterDecl.type;
-            if (importableReference) {
-                type = importableReference.typeNode;
-                importSymbols(importAdder, importableReference.symbols);
+            let type = isJs ? undefined : parameterDecl.type;

```

Comment:
`parameterDecl.questionToken` below should also be `undefined` if we're in JS.

Reply:
Oke.

Reply:
I guess Daniel merged this while I was reviewing, but @a-tarasyuk if you could fix this minor thing, that'd be great. Thanks!

Reply:
I'll fix.

Reply:
Sorry about that - good catch @gabritto!

Reply:
@gabritto Thanks